consumer-rest-core/src/main/scala/za/co/absa/spline/consumer/rest/controller/ExecutionEventController.scala

```diff
@@ -41,7 +41,7 @@ class ExecutionEventController @Autowired()(val repo: ExecutionEventRepository)
     @ApiParam(value = "Beginning of the timestamp range used for querying. If timestampStart equals 0, the service will return the first 100 execution events in database", example = "0")
     @RequestParam(value = "timestampStart", required = false, defaultValue = "0") timestampStart: Long,
     @ApiParam(value = "End of the timestamp range used for querying", example = "0")
-    @RequestParam(value = "timestampEnd", required = false, defaultValue = "0") timestampEnd: Long,
+    @RequestParam(value = "timestampEnd", required = false, defaultValue = "9223372036854775807") timestampEnd: Long,
```

Comment:
Might be better to define a value for his number or comment it.

Reply:
It's a max Long :)
I though about making it a constant, but then I'd have to extract the rest of defaults into constants. Just hesitated to do that and inlined for the sake of consistency. Let's me handle it in the scope of #351

Reply:
ok, In that case you could also do `Long.MaxValue.toString`

Reply:
Unfortunately I couldn't. The annotation value must be a constant expression. Calling `.toString` makes it dynamic.